modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx

```diff
@@ -50,104 +51,129 @@ const MDFClaimPage = ({
-				title="Total Claim"
-			>
-				<PRMFormik.Field
-					component={PRMForm.InputFile}
-					description="Upload an invoice for the Total Claim Amount"
-					displayType="secondary"
-					label="Reimbursement Invoice"
-					name="reimbursementInvoice"
-					onAccept={(value: File) =>
-						setFieldValue('reimbursementInvoice', value)
-					}
-					outline
-					small
-				/>
-
-				<ResumeCard
-					className="mb-4"
-					leftContent="Total MDF Requested Amount"
-					rightContent={getIntlNumberFormat().format(
-						values.totalrequestedAmount || 0
-					)}
-				/>
-
-				<PRMFormik.Field
-					component={PRMForm.InputCurrency}
-					description="The amount to be claimed for the Total of  selected expenses"
-					label="Total Claim Amount"
-					name="totalClaimAmount"
-					onAccept={(value: number) =>
-						setFieldValue('totalClaimAmount', value)
-					}
-					required
-				/>
-			</PRMForm.Section>
-
-			<PRMForm.Footer>
-				<div className="d-flex mr-auto">
-					<ClayButton
-						className="pl-0"
-						disabled={isSubmitting}
-						displayType={null}
-						onClick={() => onSaveAsDraft(values, formikHelpers)}
-					>
-						Save as Draft
-					</ClayButton>
-				</div>
-
-				<div>
-					<ClayButton
-						className="mr-4"
-						displayType="secondary"
-						onClick={() => onCancel()}
-					>
-						Cancel
-					</ClayButton>
-
-					<ClayButton
-						disabled={!isValid || isSubmitting || checkClaimLimit}
-						type="submit"
-					>
-						Submit
-					</ClayButton>
-				</div>
-			</PRMForm.Footer>
-		</PRMForm>
-	);
+
+					<ResumeCard
+						className="mb-4"
+						leftContent="Total MDF Requested Amount"
+						rightContent={getIntlNumberFormat().format(
+							values.totalrequestedAmount || 0
+						)}
+					/>
+
+					<PRMFormik.Field
+						component={PRMForm.InputCurrency}
+						description="The amount to be claimed for the Total of  selected expenses"
+						label="Total Claim Amount"
+						name="totalClaimAmount"
+						onAccept={(value: number) =>
+							setFieldValue('totalClaimAmount', value)
+						}
+						required
+					/>
+				</PRMForm.Section>
+
+				<PRMForm.Footer>
+					<div className="d-flex mr-auto">
+						<ClayButton
+							className="pl-0"
+							disabled={isSubmitting}
+							displayType={null}
+							onClick={() => onSaveAsDraft(values, formikHelpers)}
+						>
+							Save as Draft
+						</ClayButton>
+					</div>
+
+					<div>
+						<ClayButton
+							className="mr-4"
+							displayType="secondary"
+							onClick={() => onCancel()}
+						>
+							Cancel
+						</ClayButton>
+
+						<ClayButton
+							disabled={!isValid || isSubmitting}
+							type="submit"
+						>
+							Submit
+						</ClayButton>
+					</div>
+				</PRMForm.Footer>
+			</PRMForm>
+		);
+	};
+
+	return getClaimPage(claimsNotDraft);
```

Comment:
Since this variable can be accessed globally, I think there is no need to be passing it to the function
```suggestion
	return getClaimPage();
```